app/services/selectAccount.service.ts

```diff
@@ -0,0 +1,25 @@
+import { store } from '../contexts/FullServiceContext';
+import { selectAccountAction } from '../contexts/actions/selectAccount.action';
+import * as fullServiceApi from '../fullService/api';
+
+const selectAccount = async (accountId: string): Promise<void> => {
+  try {
+    const { account } = await fullServiceApi.getAccount({ accountId });
+
+    const { addressIds, addressMap } = await fullServiceApi.getAllAddressesForAccount({
+      accountId,
+    });
+
+    const { balance: balanceStatus } = await fullServiceApi.getBalanceForAccount({
+      accountId,
+    });
```

Comment:
None of these calls may go in parallel?

Reply:
no need for them to, they're instant communication to full service, so there's nothing to really "await"